src/sensor-overview.md

```diff
@@ -0,0 +1,47 @@
+# Thread Smart Sensor
+
+This module and submodules will walk you through how to create a Thread-enabled
+wireless sensor using Tock.
+
+## Hardware Notes
+
+To fully follow this guide you will need a hardware board that supports the
+nRF52840 microcontroller. We recommend using the nRF52840dk.
+
+Compatible boards:
+
+- nRF52840dk
+- imix
```

Comment:
imix doesn't have an nrf52840. It _may_ work with the tutorial? But it contradicts the previous statement

Reply:
Yeah that must be a copy-and-paste error.